game/menu/gamedisplay.gd

```diff
@@ -2,11 +2,15 @@ extends MarginContainer
 
 signal pressed(game_file)
 
-var game_file: ConfigFile
+class_name GameDisplay
```

Comment:
class_name belongs to the top.
https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#code-orde

Reply:
I would run the linter if it was not a python script. I mean i could pip install that, but i dont want to.